### PR TITLE
GM: Cadillac ATS Coupe 2018 support

### PR DESCRIPTION
**Pull request by vntarasov**
Remaining issues:
- resume after a full stop.

### CODE REVIEW

**Comment by rbiasini** [comment]
@vntarasov, nice. Conflicts aside, still any outstanding issues?

**Comment by andrewcremins** [comment]
This PR has been working well for me. No issues in terms of current function. Don't know if there are any any issues with any other planned GM changes.

**Comment by vntarasov** [comment]
@rbiasini no, works great!

File: selfdrive/car/gm/carstate.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def get_powertrain_can_parser(CP, canbus):
     ("LKATorqueDeliveredStatus", "PSCMStatus", 0),
   ]
 
-  if CP.carFingerprint in (CAR.VOLT, CAR.MALIBU):
+  if CP.carFingerprint == CAR.VOLT:
     signals += [
       ("RegenPaddle", "EBCMRegenPaddle", 0),
     ]
```
Comment (rbiasini) [inline selfdrive/car/gm/carstate.py]: Missing ATS in the` if` below

Comment (vntarasov) [reply selfdrive/car/gm/carstate.py]: Thanks! Fixed.

**Pull request merged**